<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramWindow</class>
 <widget class="QMainWindow" name="HistogramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>242</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Histogram</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="gbImageType">
      <property name="title">
       <string>Image type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbTargOrig">
         <property name="text">
          <string>Target Oiginal</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbTargColor">
         <property name="text">
          <string>TargetColorized</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbSource">
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbColorSpaceType">
      <property name="title">
       <string>Color space</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbRGBGrey">
         <property name="text">
          <string>RGB Grey</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbRGB">
         <property name="text">
          <string>RGB</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbLAB">
         <property name="text">
          <string>LAB</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="rbLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbFormHist">
        <property name="text">
         <string>Form histogram</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTargOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Target Original</string>
   </property>
  </action>
  <action name="actionTargColorized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Target Colorized</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="actionHistRGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionHistLAB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>LAB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
